server/chat-commands.js

```diff
@@ -662,28 +662,16 @@ const commands = {
 		target = Chat.nicknamefilter(target, user, true);
 		if (!target) return this.errorReply("Your status contains a banned word.");
 
-		let statusType = '(Online)';
-		// Should work even if users use /status with a message containing ()
-		if (user.status && user.status.includes('(') && user.status.includes(')')) {
-			statusType = user.status.slice(0, user.status.indexOf(')') + 1);
-		}
-
-		user.setStatus(`${statusType} ${target}`);
-		this.sendReply(`Your status has been set to: ${target}`);
+		user.setStatus(null, target);
```

Comment:
I don't think `null` triggers default parameters in functions.
```suggestion
		user.setStatus(undefined, target);
```